.circleci/config.yml

```diff
@@ -292,15 +298,11 @@ jobs:
               else
                   echo "Error: Found $total installed packages (max $MAX_PACKAGES).";
               fi
+
+            # Add a dogrc so we can submit metrics to datadog
+            printf "[Connection]\napikey = $DATADOG_API_KEY\nappkey =\n" > ~/.dogrc
```

Comment:
It looks like this run command is doing double duty.. it will exit with an error if there are too many packages, but it will also report that count to data dog. My initial thought is that we still want to do the former for forked PR's but not the later (hence you using the key only after exiting early), but do we need to split this command up? 

The order of these commands will mean that there is the potential that someones forked PR will pass tests, but will not when merged. This will happen when the number of packages exceeds 3050.